src/utils.ts

```diff
@@ -180,3 +180,13 @@ export function debounce<T>(
     }, ms)
   }
 }
+
+/**
+ * Helpful wrapper around `console.warn()`
+ */
+export function warn(condition: boolean, message: string): void {
```

Comment:
Maybe call it `warnWhen` so it's easier to read: `warn when condition is true -> message`.